tests/robot/1_check_airflow.robot

```diff
@@ -11,6 +11,11 @@ Connect to Airflow and check it
     :FOR    ${enclave}    IN    @{ENCLAVES}
     \  Connect to enclave Airflow     ${enclave}
         ${dags} =                   Find Airflow DAGs
-        Should not be empty         ${dags}
+        :FOR    ${dag}   IN   @{dags}
+        \   ${tasks} =              Find Airflow Tasks  ${dag}
+            :FOR    ${task}     IN      @{tasks}
+            \   ${status} =     Trigger Airflow task    ${dag}  ${task}  2018-06-01 00:00:00
```

Comment:
What if you make *execution_date* parameter optional and default value is *today*?